TACC/Core-CMS-Custom
# fix(apcd-cms): fp-1873 register form, footnotes

## Overview / Changes

- Fix order of footnotes.
- Use markup for footnotes, not unicode.﹡

<sup>﹡ Unicode footnote character sizes are rendered inconsistently with some fonts.</sup>

## Related

- [FP-1873](https://jira.tacc.utexas.edu/browse/FP-1873)

## Testing

1. Log in to CMS.
2. Load form at http://localhost:8000/register/request-to-submit/.
3. Confirm footnotes appear in form in sequential order.
4. Confirm footnotes appear beneath form in matching order.

## UI

<details><summary>Tall Screenshot <sub>with arrows pointing to changed text</sub></summary>

![fp-1884-fp-1873-form-render](https://user-images.githubusercontent.com/62723358/196766270-840ae24e-26b2-4079-acd0-063d491f4272.jpeg)

</details>
